blog/2025-10-03-pants-2-29/index.mdx

```diff
@@ -0,0 +1,45 @@
+---
+  authors: [cburroughs]
+  tags: [announcement]
+  # Date of the actual release
+  date: 2025-10-03
+---
+
+# Pants 2.29.0 is released!
+
+<CaptionedImg src={require("./splash.jpg").default}>
+  Photo by [Klares Vincent](https://unsplash.com/@byvincentk) on
+  [Unsplash](https://unsplash.com/photos/ancient-stone-aqueduct-reflected-in-calm-water-Sbhx-SzSWBw)
+</CaptionedImg>
+
+{/* truncate */}
+
+We are pleased to announce Pants [2.29.0](https://github.com/pantsbuild/pants/blob/2.29.x/docs/notes/2.29.x.md), the latest release of [Pantsbuild](https://www.pantsbuild.org/), the scalable and ergonomic build system. To update, set `pants_version = "2.29.0"` in your `pants.toml`. If you're not using Pants yet, [get started now](https://www.pantsbuild.org/2.29/docs/getting-started).
+
+Thank you to [Klaviyo](https://www.klaviyo.com/) for their Platinum tier [sponsorship](https://www.pantsbuild.org/sponsors) throughout this release!
```

Comment:
They are no longer a sponsor and have not been since 2.28.0 :( So this should be removed.